theming.js

```diff
@@ -139,25 +150,26 @@ var ThemeManager = new Lang.Class({
     },
 
     _updateDashColor: function() {
-        if (this._settings.get_boolean('custom-background-color')) {
-            let [backgroundColor, borderColor] = this._getDefaultColors();
+        let [backgroundColor, borderColor] = this._getDefaultColors();
```

Comment:
Are we sure we get the right alpha of the opaque state from getDefaultColors()? Similar to how we remove custom styles in there, maube we should also temporary remove/add the right css classes.

Reply:
> Are we sure we get the right alpha of the opaque state from getDefaultColors()?

This is perhaps something we need to double-check, but just to be clear, this is not something I have changed in this PR. While the changeset for this function looks relatively big, actually all I'm doing is moving `backgroundColor` out of the `if{}` to have it accessible to pass on to `transparency.setColor()`.

In any case, the best way to check this would be with a theme that has some transparency by default. I'll try to get back to you on this.